spec/helpers/util.js

```diff
@@ -60,6 +60,14 @@ function isIE() {
     return ((navigator.appName === 'Microsoft Internet Explorer') || ((navigator.appName === 'Netscape') && (new RegExp('Trident/.*rv:([0-9]{1,}[.0-9]{0,})').exec(navigator.userAgent) !== null)));
 }
 
+function getEdgeVersion() {
+    var match = /Edge\/(\d+)/.exec(navigator.userAgent);
+    if (match !== null) {
+        return new Number(match[1]);
```

Comment:
Ugh, I definitely used the wrong regex here (I wanted the full decimal), and the `+` type casting is definitely much cleaner.  Looks like `+` is even slightly better than `parseFloat()` beyond being more compact, since it'll return 0 if the string was empty instead of `NaN`.  Thanks for the tip.